src/tags-input.js

```diff
@@ -10,6 +10,7 @@
  *
  * @param {string} ngModel Assignable angular expression to data-bind to.
  * @param {string=} [displayProperty=text] Property to be rendered as the tag label.
+ * @param {string=} [trackProperty=text] Property to be used for tracking items in the ng-repeat.
```

Comment:
I think `keyProperty` would be more appropriate. It's an implementation detail that it's used by `ng-repeat` to track items, and the user doesn't need to know that. The same is true for the option's description.